src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -267,24 +257,20 @@ protected Task UnregisterReminderAsync(string reminderName)
         /// Specify negative one (-1) milliseconds to disable periodic signaling.</param>
         /// <returns>Returns IActorTimer object.</returns>
         protected Task<IActorTimer> RegisterTimerAsync(
-            Func<object, Task> asyncCallback,
+            string callback,
```

Comment:
Should we validate the method when the timer is created? Then you would find out you made a mistake immediately.

Reply:
Agree